src/main/scala/net/jcazevedo/moultingyaml/package.scala

```diff
@@ -63,9 +64,15 @@ package object moultingyaml {
 
   implicit class PimpedString(val string: String) extends AnyVal {
     def parseYaml: YamlValue =
-      convertToYamlValue(new Yaml().load(string))
+      parseYaml(new LoaderOptions())
+
+    def parseYaml(loaderOptions: LoaderOptions) =
```

Comment:
I prefer not exposing SnakeYAML entities in MoultingYAML's public API. What do you think of changing this (and `parseYamls` below to something with a signature of `parseYaml(allowDuplicateKeys: Boolean, allowRecursiveKeys: Boolean, maxAliasesForCollections: Int)`?

Reply:
Seems like a good idea, I'll do that and add default values similar to what SnakeYAML has.